conf.d/mysql.yaml.example

```diff
@@ -11,8 +11,23 @@ instances:
     #   - optional_tag1
     #   - optional_tag2
     # options:               # Optional
-    #   replication: 0
-    #   galera_cluster: 1
+    #   replication: false
+    #   galera_cluster: false
+    #   extra_status_metrics: true
+    #   extra_innodb_metrics: true
+    #   extra_performance_metrics: true
+    #   schema_size_metrics: false
+    #
+    #     NOTE: extra_performance_metrics and schema_size_metrics will run two heavier queries
+    #           against your DB to compute the relevant metrics for all your existing schemas.
+    #           Due to the nature of these calls, if you have a high number of tables and/or schemas,
+    #           you may be subject to some negative impact in the performance of your DB.
+    #           Please bear that in mind when enabling them.
```

Comment:
Let's just mention here that `extra_performance_metrics` will only work when `performance_schema` is enabled and on versions >= `5.6.0`